test/dbus/lvmdbustest.py

```diff
@@ -1832,6 +1832,37 @@ def test_external_pv_create(self):
 		cmd = ['pvcreate', target.Pv.Name]
 		self._verify_existence(cmd, cmd[0], target.Pv.Name)
 
+	def _create_nested(self, pv_object_path):
+		vg = self._vg_create([pv_object_path])
+		pv = ClientProxy(self.bus, pv_object_path, interfaces=(PV_INT,))
+
+		self.assertEqual(pv.Pv.Vg, vg.object_path)
+		self.assertTrue(pv_object_path in vg.Vg.Pvs)
```

Comment:
There's also ``assertIn`` which provides a better error message than just ``False is not True`` in case of failure. Actually, there are quite many ``assert`` functions in the *unittest* module -- https://docs.python.org/2/library/unittest.html#assert-methods